gamezilla/urls.py

```diff
@@ -39,7 +40,7 @@
     # You MUST use an empty string as the URL prefix
     path("", include("pwa.urls")),
     path("new-home/",game_views.new_home),
-    path("profile/",game_views.profile)
```

Comment:
Please remove this view from ``profile`` app

Reply:
25//`from account import views as game_views`

42//`path("profile/",game_views.profile)`

like this?

Reply:
No, 
Actually, you need to remove this view.
https://github.com/niteshsinha17/GameZilla/blob/b97854b38a5f5b98f1bb439fdcf530d5722b6a4d/game/views.py#L26-L28